bbp/nse/bluepyopt/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  config,
+  fetchFromGitHub,
+  pythonPackages
+}:
+
+pythonPackages.buildPythonPackage rec {
+    pname = "BluePyOpt";
+    version = "1.6dev";
+    name = "${pname}-${version}";
+
+    src = fetchFromGitHub {
```

Comment:
why not fetchPypi?

Reply:
I tend to trust more github than anything else to keep old version and archive properly.  Do you see it as an issue ? Is there version released on Pypi and not tagged on github ?

Reply:
I see an issue (even if a minor one) with using two approaches for apparently same thing.
For `fetchPypi` one we have a benefit of not having to maintain version-commit correspondence manually and thus having guaranteed same versioning in virtualenv and Nix worlds, and I find it useful; but if trust in GitHub is more important, `fetchFromGitHub` should be preferred in other cases as well.

Reply:
Thx, I think I will keep Github for now and see with Werner for the testing

Reply:
Ok, but then at least update the version number to the correct one ...
i.e. 1.6.27